test/compress/big_int.js

```diff
@@ -0,0 +1,16 @@
+bigint_support: {
+    input: {
+        100000n;
+        -15n;
+        0x20n;
+        0b101n;
+        0o7n;
+    }
+    expect: {
+        100000n;
+        -15n
+        0x20n;
+        0b101n;
+        0o7n;
+    }
```

Comment:
This needs to be expect_exact because the expect tests compare the Terser AST objects and not the real output. expect_exact actually forces the output code to run.

Reply:
Hmm.. got it! Will change ASAP